frontend/src/styles/utils/decorations.scss

```
@@ -33,6 +33,50 @@ body {
   &-hidden { overflow: hidden; }
 }
 
+/////////////////////////
+//   BORDER CLASSES    //
+/////////////////////////
+$spacing-xxs:  0.1px;
+$spacing-xs:  0.5px;
+$spacing-s:   1px;
+$spacing-m:   2px;
+$spacing-l:   4px;
+$spacing-xl:  8px;
+
+// border-radius
+.border{
+  &-m{
+    border-radius: $spacing-m $spacing-m $spacing-m $spacing-m;
```

I think its fine to just use `3px` here instead of the spacing classes - we won't be seeing a lot of border radius variations

```suggestion
    border-radius: 3px;
```